modules/ipc/ipcProviderBackend.js

```diff
@@ -593,4 +597,24 @@ module.exports = function(){
     ipc.on('ipcProvider-writeSync', function(event, payload){
         sendRequest(event, payload, true);
     });
-};
\ No newline at end of file
+
+    const uiLoggers = {};
+
+    ipc.on('console_log', function(event, id, logLevel, logItemsStr) {
```

Comment:
This whole stuff belongs not here, but in: https://github.com/ethereum/mist/blob/master/modules/ipcCommunicator.js